examples/example.html

```diff
@@ -186,12 +186,12 @@
 
       timelineRect();
       timelineCircle();
-      timelineHover();
-      timelineStackedIcons();
-      timelineLabelColor();
-      timelineRotatedTicks();
-      timelineRectColors();
-      timelineRelativeTime();
+      //timelineHover();
+      //timelineStackedIcons();
+      //timelineLabelColor();
+      //timelineRotatedTicks();
+      //timelineRectColors();
+      //timelineRelativeTime();
```

Comment:
Sorry, I was very premature with my pull request.  Please allow me to clean it up and updated the README.md file.  I will then resubmit.